sql/branch.sql

```diff
@@ -0,0 +1,16 @@
+BEGIN;
+    CREATE TYPE email_auth_intent AS ENUM ('sign-in');
```

Comment:
The purpose of storing this is that for `sign-in`, we have to make sure that an email reference (to the `emails` table) is maintained. When an email is deleted, this record should be deleted too. (Which reminds me, got to add a test for that) This prevents the case where a user mistakenly adds an email address, and an attacker verifies it and uses it to log in. (Not sure how likely it is, but plausible)